### ISSUE TITLE
Configure sentry for backend

### ISSUE THREAD

**Issue by nicksnyder** [sourcegraph/sourcegraph#2447]
Our sentry project has two teams: #web and #sourcegraph.

The #sourcegraph team should turn into the "backend" team with appropriate teammates (e.g. not me, not Quinn).

**Comment by felixfbecker** [comment]
We also might need to separate frontend and backend projects, and separate dogfood and prod with environments instead. Will require adding a site config to set a separate DSN.

**Comment by keegancsmith** [comment]
We can also start getting more granularity. For example @tsenart and I can create a project for repo-updater or core services / etc.

**Comment by felixfbecker** [comment]
@keegancsmith Yeah, just consider it needs one sentry DSN for each (for customers too). I don't care as long as someone also owns the rest of the backend errors

**Comment by nicksnyder** [comment]
@slimsag can you put this into 3.2 milestone or backlog depending on whether you think this will get done in March or not?
